ext/opcache/ZendAccelerator.c

```diff
@@ -129,7 +129,7 @@ static ZEND_INI_MH((*orig_include_path_on_modify)) = NULL;
 static zend_result (*orig_post_startup_cb)(void);
 
 static zend_result accel_post_startup(void);
-static zend_result accel_finish_startup(void);
+static int accel_finish_startup(void);
```

Comment:
`zend_result` should be kept

Reply:
Agree, it's not a big problem to keep ``zend_result``, especially if this is ``static`` and not exposed.